stdlib/public/SDK/CoreFoundation/CoreFoundation.swift

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Please name this file something more descriptive than CoreFoundation.swift. CFObject.swift maybe.